docs/content/preview/yugabyte-platform/security/ldap-authentication-platform.md

```diff
@@ -20,9 +20,9 @@ type: docs
   </li>
 </ul>
 
-LDAP Authentication is similar to password authentication, except that it uses the LDAP protocol to verify the password. Therefore, before LDAP can be used for authentication, the user must already exist in the database and have appropriate permissions.
+LDAP Authentication in YugabyteDB is similar to password authentication, except that it uses the LDAP protocol to verify the password. Therefore, before LDAP can be used for authentication, the user must already exist in the database and have appropriate permissions.
```

Comment:
LDAP Authentication in YugabyteDB  -->  LDAP authentication in YugabyteDB Anywhere

Reply:
The settings on this page may be set in YBA, but the settings affect the database,  not YBA (YBA auth is covered at https://docs.yugabyte.com/preview/yugabyte-platform/administer-yugabyte-platform/ldap-authentication/).  I put "in YugabyteDB" purposely make this distinction.